lib/nodejs/lib/thrift/connection.js

```diff
@@ -209,10 +205,9 @@ Connection.prototype.connection_gone = function () {
   this.connected = false;
   this.ready = false;
 
+  this.retry_delay = Math.floor(this.retry_delay * this.retry_backoff);
```

Comment:
Is there good reason to change the initial timeout value and also `retry_backoff` 1.7 -> 1.5 ?